oc.mk

```diff
@@ -25,7 +25,7 @@ define oc_project
 endef
 
 define oc_validate
-	$(OC) process -f $(1) --local $(2) \
+	$(OC) process --ignore-unknown-parameters=true -f $(1) --local $(2) \
```

Comment:
@matthieu-foucault one little quirk of the `oc` client, whenever there's a boolean attribute you don't have to write `=true` as passing `--ignore-unknown-paramaters` implies true since the default is false (no super important, merged as-is)